code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -0,0 +1,118 @@
+//Cardboard cutouts! They're man-shaped and can be colored with a crayon to look like a human in a certain outfit, although it's limited, discolored, and obvious to more than a cursory glance.
+/obj/item/cardboard_cutout
+	name = "cardboard cutout"
+	desc = "A vaguely humanoid cardboard cutout. It's completely blank."
+	icon = 'icons/obj/cardboard_cutout.dmi'
+	icon_state = "cutout_basic"
+	w_class = 4
+	var/list/possible_appearances = list("Assistant", "Clown", "Mime", "Traitor", "Nuke Op", "Cultist", "Clockwork Cultist", "Revolutionary", "Wizard", "Shadowling", "Xenomorph", "Swarmer", \
+	"Ash Walker", "Deathsquad Officer", "Ian") //Possible restyles for the cutout; add an entry in change_appearance() if you add to here
+	var/pushed_over = FALSE //If the cutout is pushed over and has to be righted
+	var/deceptive = FALSE //If the cutout actually appears as what it portray and not a discolored version
+
+/obj/item/cardboard_cutout/attack_hand(mob/living/user)
+	if(user.a_intent == "help" || pushed_over)
+		return ..()
+	user.visible_message("<span class='warning'>[user] pushes over [src]!</span>", "<span class='danger'>You push over [src]!</span>")
+	playsound(src, 'sound/weapons/Genhit.ogg', 50, 1)
+	name = initial(name)
+	desc = "[initial(desc)] It's been pushed over."
+	icon_state = "cutout_pushed_over"
+	color = initial(color)
+	pushed_over = TRUE
+
+/obj/item/cardboard_cutout/attack_self(mob/living/user)
+	if(!pushed_over)
+		return
+	user << "<span class='notice'>You right [src].</span>"
+	desc = initial(desc)
+	icon_state = initial(icon_state) //This resets a cutout to its blank state - this is intentional to allow for resetting
+	pushed_over = FALSE
+
+/obj/item/cardboard_cutout/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon))
+		change_appearance(I, user)
+	else
+		return ..()
+
+/obj/item/cardboard_cutout/proc/change_appearance(obj/item/toy/crayon/crayon, mob/living/user)
+	if(!crayon || !user)
+		return
+	if(pushed_over)
+		user << "<span class='warning'>Right [src] first!</span>"
+		return
+	var/new_appearance = input(user, "Choose a new appearance for [src].", "26th Century Deception") as null|anything in possible_appearances
+	if(!new_appearance || !crayon || !user.canUseTopic(src))
+		return
+	user.visible_message("<span class='notice'>[user] gives [src] a new look.</span>", "<span class='notice'>Voila! You give [src] a new look.</span>")
+	crayon.use_charges(1)
+	crayon.check_empty(user)
+	if(!deceptive)
+		color = "#FFD7A7"
+	switch(new_appearance)
+		if("Assistant")
+			name = "[pick(first_names_male)] [pick(last_names)]"
```

Comment:
Yeah. Is ugly, discoloured, flimsy cardboard only good enough for one and not the other?